setup.py

```diff
@@ -69,5 +69,9 @@ def get_long_description():
         package_dir={'': 'src'},
         install_requires=INSTALL_REQUIRES,
         extras_require=EXTRAS_REQUIRE,
-        tests_require=TESTS_REQUIRE
+        tests_require=TESTS_REQUIRE,
+        entry_points='''
+            [console_scripts]
+            pushover=pushover_complete.pushover_cli:cli
+        '''
```

Comment:
I've never understood why people recommend this stupid literal string ini-like configuration. It's a dict:

```python
entry_points={
    'console_scripts': ['pushover=pushover_complete.pushover_cli:cli']
}